package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "package.json",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "task1": "nodemon ./src/task1.js",
+    "task2": "nodemon ./src/task2.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/IRYNA-MAZURA/NodeJS_MP_homework_1.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/IRYNA-MAZURA/NodeJS_MP_homework_1/issues"
+  },
+  "homepage": "https://github.com/IRYNA-MAZURA/NodeJS_MP_homework_1#readme",
+  "dependencies": {
+    "@babel/polyfill": "^7.12.1",
+    "csvtojson": "^2.0.10"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.13.10",
+    "@babel/core": "^7.13.10",
+    "@babel/preset-env": "^7.13.10"
+  }
+}
```

Comment:
A small proposal - node.js version is often specified in the engines section inside package.json.
[Check the link!](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines)